packages/web/components/table/assets-table.tsx

```diff
@@ -278,6 +282,29 @@ export const AssetsTable: FunctionComponent<Props> = observer(
       ? filteredSortedCells
       : filteredSortedCells.slice(0, 10);
 
+    const favoritesData: TableCell[] = useMemo(() => {
+      return tableData
+        .map((coin) => {
+          if (favoritesList.includes(coin.coinDenom)) {
+            coin.isFavorite = true;
+            coin.onToggleFavorite = () => {
+              const newFavorites = favoritesList.filter(
+                (d) => d !== coin.coinDenom
+              );
+              onSetFavoritesList(newFavorites);
+            };
+          } else {
+            coin.isFavorite = false;
+            coin.onToggleFavorite = () => {
+              const newFavorites = [...favoritesList, coin.coinDenom];
+              onSetFavoritesList(newFavorites);
+            };
+          }
+          return coin;
+        })
+        .sort((a, b) => Number(b.isFavorite) - Number(a.isFavorite));
```

Comment:
See video on sorting. Perhaps you could filter the favorites from `tableData`, then return a new list with `favoritesList.concat(favoritesFilteredTableData)` or similar.

IMO:
* Sorting: favorites always at top.
* Search query: favorites always at top but filtered by query.

Reply:
Fixed this in most recent commit